Gemfile

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
-source "https://rubygems.org"
+source 'https://rubygems.org'
 gem 'fastlane', '~> 2'
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 7.0'
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 9.0'
```

Comment:
What do you think of `~> 9.1` give that the version resolved now?

Reply:
Addressed in https://github.com/Automattic/simplenote-android/pull/1616/commits/305de22d93d56b6a72e36ebd5ace724c238ab37a